t/50-npg_pipeline-daemon-analysis.t

```diff
@@ -33,6 +33,8 @@ local $ENV{PATH} = join q[:], $temp_directory, $current_dir.'/t/bin', $ENV{PATH}
 
 my $dbic_util = t::dbic_util->new();
 my $schema = $dbic_util->test_schema();
+my $test_run_1235 = $schema->resultset(q[Run])->find(1235);
+$test_run_1235->update_run_status('analysis pending', 'pipeline',);
```

Comment:
As I get it from the fixtures this will only be testing `no_auto_analysis` and not testing `no_auto`. I think is best if we test both paths.

If I understand correctly the idea will be to check that by introducing the 2 new `no auto` tags for analysis but not setting it for any other runs, you expect everything to stay the same. Then this run 1235 is used to test a run with the `no_auto_analysis` tag will not go into analysis. Which I guess is kind of fair but it was difficult to understand.

If you keep things as they are now you may also need to change some other lines se line 94 `ok(scalar @test_runs >= 3, 'at least three runs are analysis pending');` which I guess should become `>= 4`. Line 95 should include 1235. And so on in the file. 

So maybe easier to create a new subtest and do run + tag 52 produces no change in status, run + tag 53 produces no change in status and run without tag gets status change. Otherwise you may end having to re-check half of the old test code which seems as an overkill for your change.